common_benchmark_suite/openxla/benchmark/models/jax/t5/t5_for_conditional_generation.py

```diff
@@ -32,26 +34,20 @@ def __init__(self, batch_size: int, seq_len: int, model_name: str):
         "padding": True,
         "return_tensors": "jax",
     }
+    self.generation_config = GenerationConfig.from_pretrained(
+        model_name, max_new_tokens=max_new_tokens)
 
   def generate_default_inputs(self) -> str:
     return "summarize: My friends are cool but they eat too many carbs."
 
-  def preprocess(self, input_text: str) -> Tuple[Any, Any]:
+  def preprocess(self, input_text: str) -> Tuple[Any,]:
     batch_input = [input_text] * self.batch_size
     inputs = self.tokenizer(batch_input, **self.tokenization_kwargs)
-    return (inputs["input_ids"], inputs["attention_mask"])
+    return (inputs["input_ids"],)
```

Comment:
I think we can write
```suggestion
    return inputs["input_ids"]
```
The tool should support the single value type.

Reply:
Thanks, that works